XPlot.DotNet.Interactive.KernelExtensions/XPlotKernelExtension.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Interactive.Rendering;
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using XPlot.Plotly;
+using static Microsoft.DotNet.Interactive.Rendering.PocketViewTags;
+
+namespace XPlot.DotNet.Interactive.KernelExtensions
+{
+    public class XPlotKernelExtension : IKernelExtension
+    {
+        public Task OnLoadAsync(IKernel kernel)
+        {
+            Formatter<PlotlyChart>.Register((chart, writer) =>
+           {
+               writer.Write(GetChartHtml(chart));
+           }, "text/html");
+
+            return Task.CompletedTask;
+        }
+
+        private string GetChartHtml(PlotlyChart chart)
```

Comment:
Because this function is just chart -> string, I suspect you can write a really precise unit test covering it

Reply:
And then your kernel level tests just need to test the actual integration

Reply:
Thanks I did this in the commit here - https://github.com/dotnet/try/pull/365/commits/29fad696321476bed1b5ce5efc442921f2e63ae7